crates/subspace-networking/src/request_handlers/pieces_by_range.rs

```diff
@@ -28,14 +17,14 @@ pub struct PiecesToPlot {
 #[derive(Debug, Clone, Eq, PartialEq, Encode, Decode)]
 pub struct PiecesByRangeRequest {
     /// Start of the requested range
-    pub from: PieceIndexHash,
+    pub start: PieceIndexHash,
     /// End of the requested range
-    pub to: PieceIndexHash,
+    pub end: PieceIndexHash,
 }
 
 impl GenericRequest for PiecesByRangeRequest {
-    const PROTOCOL_NAME: &'static str = PROTOCOL_NAME;
-    const LOG_TARGET: &'static str = LOG_TARGET;
+    const PROTOCOL_NAME: &'static str = "/subspace/sync/pieces-by-range/0.1.0";
```

Comment:
It's inconsistent with `object-mappings` handler protocol constansts. Also, it's easier to read protocol name at the beginning of the file.

Reply:
This constant is not used anywhere else, it kinda confuses because it assumes that it will be shared and it has other applications. I think it is better to inline them here.

As for consistency, I can also inline those in object-mappings.